recipes/clangdev/build.sh

```diff
@@ -0,0 +1,16 @@
+mkdir build
+cd build
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DLLVM_ENABLE_RTTI=ON \
+  -DBUILD_SHARED_LIBS=OFF \
+  -DLLVM_INCLUDE_TESTS=OFF \
+  -DLLVM_INCLUDE_UTILS=OFF \
+  -DLLVM_INCLUDE_DOCS=OFF \
+  -DLLVM_INCLUDE_EXAMPLES=OFF \
+  ..
+
+make -j 8
```

Comment:
Yep, I think are comments got interwoven so I missed your response. I completely agree. Maybe we can compile a hello world style program in the test step instead.